projects/storefrontlib/src/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -70,8 +74,17 @@ export class TabParagraphContainerComponent
     )
   );
 
-  select(tabNum: number): void {
-    this.activeTabNum = tabNum;
+  select(tabNum: number, event): void {
```

Comment:
1. Can we add type for `event`?
2. For backward compatibility, `event` should be an optional parameter and the implementation of the method should take it into account it can be null.